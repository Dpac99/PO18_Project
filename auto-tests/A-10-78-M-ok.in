100100
1
e1.obj
4
5
D1
P4
0
2
0
