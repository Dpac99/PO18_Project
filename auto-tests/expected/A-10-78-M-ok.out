Identificador: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
4 - Portal do Docente
0 - Sair
Escolha uma opção: Ficheiro a abrir: Pode preencher inquérito do projecto P5 da disciplina D1
Resultados do inquérito do projecto P4 da disciplina D1
Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
4 - Portal do Docente
0 - Sair
Escolha uma opção: Portal do Docente
1 - Criar projecto
2 - Fechar projecto
3 - Ver submissões de um projecto
4 - Ver alunos de disciplina leccionada
5 - Ver resultados de um inquérito
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P4
 * Número de submissões: 0
 * Número de respostas: 0
 * Tempos de resolução (horas) (mínimo, médio, máximo): 0, 0, 0
Portal do Docente
1 - Criar projecto
2 - Fechar projecto
3 - Ver submissões de um projecto
4 - Ver alunos de disciplina leccionada
5 - Ver resultados de um inquérito
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
4 - Portal do Docente
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
4 - Portal do Docente
0 - Sair
Escolha uma opção: 
